freemocap/core_processes/post_process_skeleton_data/process_single_camera_3d_data.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import logging
+
+import numpy as np
+
+from freemocap.core_processes.capture_volume_calibration.triangulate_3d_data import save_mediapipe_skeleton_data_to_npy
+from freemocap.utilities.project_3d_data_to_z_plane import project_3d_data_to_z_plane
+
+logger = logging.getLogger(__name__)
+
+
+def process_single_camera_skeleton_data(input_image_data_frame_marker_xyz: np.ndarray, raw_data_folder_path: Path) \
```

Comment:
```suggestion
def process_single_camera_skeleton_data(input_image_data_frame_marker_xyz: np.ndarray, raw_data_folder_path: Union[str, Path], project_to_z_plane:bool=True):
```

1- Always want to accept  `str` for paths b/c that's the primitive, but I like to allow for `Path`s too (though that is baking in our choice to use `pathlib` - many use `os.path`) - You'll probably need to import `Union` from `Typing`

2 - gonna put the Z-plane stuff behind a bool that's `True` by default so we can turn the projection off later if we want to